src/main/java/core/DriverManager.java

```diff
@@ -0,0 +1,47 @@
+package core;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * Manages and initializes the browser drivers.
+ */
+public class DriverManager {
+
+    public static WebDriver driver = null;
+    public static String browser;
+
+    private DriverManager() {
```

Comment:
I think this singleton should be modified and return only the instance of driver, like we saw in the class today

Reply:
Yeah, I already changed it